crypto-ffi/src/core_crypto/logger.rs

```diff
@@ -276,16 +276,16 @@ pub fn set_max_log_level(level: CoreCryptoLogLevel) {
     log::set_max_level(level.into());
 }
 
+/// Set the CC logger
 #[cfg(target_family = "wasm")]
 #[wasm_bindgen]
-impl CoreCryptoFfi {
-    /// Set the CC logger
-    pub fn set_logger(logger: CoreCryptoLogger) {
-        set_logger_only_inner(logger);
-    }
+pub fn set_logger(logger: CoreCryptoLogger) {
+    set_logger_inner(logger);
+}
```

Comment:
So at this point, we have the wasm-only `fn set_logger`, and the non-wasm `fn set_logger`, which differ only in their annotations. So really it should be all one declaration now:

```rust
/// Set the CC logger
#[cfg_attr(target_family = "wasm", wasm_bindgen)]
#[cfg_attr(not(target_family = "wasm"), uniffi::export]
pub fn set_logger(logger: CoreCryptoLogger) {
  // contents of set_logger_inner go here
}
```

Reply:
I agree, but the logger type differs between uniffi and wasm.  `setLogger`uses Arc<CoreCryptoLogger>. For `set_max_log_level` we can do it like that.